### DISCUSSION TITLE
Can't buy a domain due to price mismatch error.

### DISCUSSION THREAD

**Discussion by srcnalt** [vercel/community#4433] [Bug Report]
### Page to Investigate

https://vercel.com/domains

### Steps to Reproduce

Sometimes the domain you want to buy shows a different renewal price (higher) and even if you accept it does not work due to price mismatch.

For example, you can try `vercel.cat`, the price is 20$, renewal will be shown 50$ and when clicked continue you will receive price mismatch error.

Still haven't received a reply from Vercel support as a pro user.

**Comment by swarnava** [comment]
Hi,

Are you buying the domain programmatically using API? Can you share a screencast how you are reproducing this?

**Comment by srcnalt** [comment]
@swarnava attempting to buy it via Vercel interface, here is how it looks:
![Screenshot 2023-10-01 235059](https://github.com/vercel/community/assets/3163281/c0bd111e-436f-4743-ab8c-8b554cf584d1)
![Screenshot 2023-10-10 145506](https://github.com/vercel/community/assets/3163281/0766a122-29f0-4893-aa73-a2a63743b943)

**Reply by srcnalt** [reply]
Just noticed the problem above says `domain_already_exists`, here is how the domain I want to buy looks with the error. First image looks the same.
![Screenshot 2023-10-10 151508](https://github.com/vercel/community/assets/3163281/89ee9d96-c08c-42f5-b297-43c1e833f516)

**Reply by srcnalt** [reply]
@swarnava do these help?

**Reply by swarnava** [reply]
Hi,

This is likely because I tried to purchase `vercel.cat` domain and couldn't reproduce. Are you able to reproduce with other domain as well?

**Reply by srcnalt** [reply]
I was trying to buy neocortex.link, hope I still can 🤷

**Reply by srcnalt** [reply]
Btw, make sure not to buy it 😄 I still want this address.

**Reply by srcnalt** [reply]
same issue is there with ncx.link as well

**Reply by swarnava** [reply]
Hi, Can you share a screenshot ?

**Reply by srcnalt** [reply]
I can do better, here is a video

https://github.com/vercel/community/assets/3163281/c8729f1a-b49e-41b1-91c0-f5d3eeb28bfb

**Comment by srcnalt** [comment]
Good morning, day 3, the issue still persists.

**Comment by srcnalt** [comment]
Good morning, day 4, the issue still persists.

**Comment by srcnalt** [comment]
Good morning, day 5, the issue still persists.
***
Though I got a reply from support, citing this discussion
> This has been escalated to the Engineering team to fix and investigate and we will provide further details on that as and when they become available.

**Comment by srcnalt** [comment]
Good morning, day 6, the issue still persists.

**Comment by srcnalt** [comment]
G'day 🤠 ! Day 7, the issue still persists. One big week! 🎉 
***
Edit: I got message from support about the details of the issue and will wait for the fix now, no more nagging 😄 
***
I got direct support from Vercel and the domain was then transferred to my account by them.
